<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_num_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_num_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_done">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_num</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_to_up">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>up</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_to_control</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_to_down">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>down</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_to_control</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_to_left">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>185</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>left</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_to_control</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_to_right">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>185</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>right</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttons_to_control</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>20</y>
      <width>250</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_to_card">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>185</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Chose card</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttons_num"/>
  <buttongroup name="buttons_to_control"/>
 </buttongroups>
</ui>
